Client/Frontend/Strings.swift

```diff
@@ -183,8 +183,13 @@ extension String {
                 value: "Toolbar Settings",
                 comment: "Contextual hints are little popups that appear for the users informing them of new features. This one is a call to action for the popup describing search bar placement. It indicates a user can navigate to the settings page that allows them to customize the placement of the search bar.",
                 lastUpdated: .v98)
+            public static let SearchBarTopPlacement = MZLocalizedString(
+                "ContextualHints.Toolbar.Top.Description",
```

Comment:
I think the version number should be in the key as well. Correct me if I am wrong @electricRGB

Reply:
Oh that's right! Forgot that - I'll update :)